### PR TITLE
 Cephfilesystem creation test based on bug

### PR DESCRIPTION
**Pull request by shivamdurgbuns**
Adding test to see if the cephfilesystem can not be created with the same name if it already exists
bug: https://bugzilla.redhat.com/show_bug.cgi?id=1969216

Polarian: [OCS-5793](https://polarion.engineering.redhat.com/polarion/#/project/OpenShiftContainerStorage/workitem?id=OCS-5793)

### CODE REVIEW

File: tests/internal/test_cephfilesystem_creation.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+
+class TestCephFileSystemCreation(ManageTest):
+    """
+    Testing Creation of a filesystem and checking its existence.
+    Also checking if the same filesystem can't be created twice.
+    """
+
+    @tier2
+    @green_squad
```
Comment (ramkiperiy) [inline tests/internal/test_cephfilesystem_creation.py:24]: You need to add this TC in polarion and update the polarion ID here.

File: ocs_ci/helpers/helpers.py [hunk 2]
```diff
@@ -605,30 +605,39 @@ def create_ceph_block_pool(
...
+    cephfs_data["metadata"]["namespace"] = namespace
+    if label:
+        cephfs_data["metadata"]["labels"] = label
+
+    try:
+        cephfs_data = create_resource(**cephfs_data)
+        cephfs_data.reload()
+    except Exception as e:
+        logger.error(e)
+        raise e
```
Comment (ramkiperiy) [inline ocs_ci/helpers/helpers.py:635]: Hope this part of code is validated with failure message, if required do add required exception message.

Comment (shivamdurgbuns) [reply ocs_ci/helpers/helpers.py:635]: this is added if any unknown error has is thrown by create resource function while creating the object

File: tests/internal/test_cephfilesystem_creation.py (hunk 1 above)
Comment (shivamdurgbuns) [reply tests/internal/test_cephfilesystem_creation.py:25]: added polarian id

File: ocs_ci/helpers/helpers.py [hunk 3]
```diff
@@ -605,30 +605,39 @@ def create_ceph_block_pool(
     return cbp_obj
 
 
-def create_ceph_file_system(pool_name=None):
+def create_ceph_file_system(
+    cephfs_name=None, label=None, namespace=constants.OPENSHIFT_STORAGE_NAMESPACE
```
Comment (ramkiperiy) [inline ocs_ci/helpers/helpers.py:609]: Have we checked regression for this? i.e. Is there any TC which is consuming this function with pool_name param defined in the function call?

Comment (ramkiperiy) [reply ocs_ci/helpers/helpers.py:609]: Got clarity from Shivam that its validated and based on the output added Verified label.

**Review by ramkiperiy — APPROVED** [review]

**Review by pintojoy — APPROVED** [review]
LGTM

**Pull request merged**